src/lib/viewers/controls/media/MediaSettingsControls/MediaSettingsMenuItem.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import classNames from 'classnames';
+import IconArrowRight24 from '../../icons/IconArrowRight24';
+import './MediaSettingsMenuItem.scss';
+
+export type Props = {
+    className?: string;
+    label: string;
+    onClick: () => void;
+    value: string;
+};
+
+export default function MediaSettingsMenuItem({ className, onClick, label, value }: Props): JSX.Element {
+    return (
+        // TODO: handle all keyboard events
+        // eslint-disable-next-line jsx-a11y/click-events-have-key-events
```

Comment:
Does `onClick` automatically support "enter" key presses for non-interactive elements? If not, it seems like we should add it.